DS/DSAssignment2/Stack.java

```diff
@@ -0,0 +1,117 @@
+import java.util.Scanner;
+
+
+public class Stack {
+private int top;
+private int size;
+Node<Object> node;
+
+
+
+public int getTop() {
+	return top;
+}
+
+public void setTop(int top) {
+	this.top = top;
+}
+
+public int getSize() {
+	return size;
+}
+
+public void setSize(int size) {
+	this.size = size;
+}
+
+public Node<Object> getNode() {
```

Comment:
ACTION: Why is this method required and that too 'public'? Not necessary to have getters/setters for every private field.